helm/cluster-resources/templates/coredns-np.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: coredns
+  namespace: kube-system
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  egress:
+  - {}
+  ingress:
+  - ports:
+    - port: 1053
+      protocol: UDP
+    - port: 1053
+      protocol: TCP
+    - port: 53
+      protocol: UDP
+    - port: 53
+      protocol: TCP
+  - ports:
```

Comment:
Why two separate blocks for these ports?

Reply:
I copied the rendered manifest https://github.com/giantswarm/coredns-app/blob/master/helm/coredns-app/templates/np.yaml